frontend/packages/topology/src/components/quick-search/QuickSearchDetails.tsx

```diff
@@ -26,6 +28,11 @@ const QuickSearchDetails: React.FC<QuickSearchDetailsProps> = ({ selectedItem, c
         className="odc-quick-search-details__form-button"
         onClick={(e) => {
           handleCta(e, selectedItem, closeModal);
+          fireTelemetryEvent('Quick Search Used', {
+            id: selectedItem.uid,
+            type: selectedItem.type,
+            name: selectedItem.name,
```

Comment:
The same event is always fired immediately after `handleCta` utility function is called. Maybe you can pass the `fireTelemtryEvent` function to `handleCta` and it can have a single implementation for firing the `Quick Search Used` event?

Reply:
I tried that before I created `withTelemetry` HOC, and it has some issues. Will try it again after @vojtechszocs 's fix on dynamic-demo-plugin folder. (both `yarn build` and `yarn build-dev` don't work for me)

Reply:
It's fine for now, but it shouldn't have been an issue to pass in a function reference parameter to `handleCta`.